sdk/ml/azure-ai-ml/tests/test_configs/compute/compute-ci-custom-app.yaml

```diff
@@ -0,0 +1,27 @@
+name: banchci
+type: computeinstance
+size: STANDARD_DS3_V2
+
+custom_applications:
+    - name: rstudio-workbench
+      image:
```

Comment:
There is top level type property which is missing. https://github.com/Azure/azureml_run_specification/blob/6daeac53aec1575d41f98b0698211df957bf52e8/configs/compute/my-ComputeInstance.yaml#:~:text=type%3A%20%22-,docker,-%22

Although this is not currently supported in backend but we have a feature in pipeline that will require this.

Reply:
If the property is not supported in the backend, it would confuse users to provide it on the SDK right now. It is not a breaking change to add a new property, so why not wait until the backend is ready?

Reply:
The parameter has been added now.